src/pages/SimpleViewer.jsx

```diff
@@ -127,7 +153,21 @@ export default function SimpleViewer(props) {
                 omitHeader
               />
               {user && (
-                <button type="button" onClick={uploadMessage}>Upload</button>
+                <>
+                  <Button
+                    onClick={uploadMessage}
+                    variant="contained"
+                    style={{ marginRight: 10 }}
+                  >
+                    Upload
+                  </Button>
+                  <Button
+                    onClick={askQuestion}
+                    variant="contained"
+                  >
+                    Ask ARA
```

Comment:
I tried the Ask ARA functionality with the covidkop response. No results or knowledge_graph were returned and the error message was "Unable to parse message". I think we should change that error to be more descriptive when there are no answers returned. My thinking is that we parse those messages without error and then display an error message in the AnswersetView.

Reply:
Opened issue #40